src/microbe_stage/Microbe.Contact.cs

```diff
@@ -1500,17 +1500,8 @@ private void IngestEngulfable(IEngulfable target, float animationSpeed = 2.0f)
         target.HostileEngulfer.Value = this;
         target.PhagocytosisStep = PhagocytosisPhase.Ingestion;
 
-        // Disable collisions
-        body.CollisionLayer = 0;
-        body.CollisionMask = 0;
-
         body.ReParentWithTransform(this);
 
-        var originalRenderPriority = target.RenderPriority;
```

Comment:
It seems the original render priority no longer gets stored and this logic is also different. Why does this need to change to make cilia pull work?

Reply:
Have you actually seen this?
https://github.com/Revolutionary-Games/Thrive/blob/006237d9908dd276daefc0c16b1854343b83c410/src/microbe_stage/Microbe.Contact.cs#L1549-L1561
The logic does not change. Just the ordering of statements are sorted properly now and the original render priority are now _stored_ in the EngulfedObject instance and to make the following possible:

https://github.com/Revolutionary-Games/Thrive/blob/006237d9908dd276daefc0c16b1854343b83c410/src/microbe_stage/Microbe.Contact.cs#L1879-L1881

Reply:
I see, now that you point these out, I think I understand this change now.